js/src/logger.ts

```diff
@@ -582,26 +584,20 @@ export class Logger<IsAsyncFlush extends boolean> {
    * @param event.metrics: (Optional) a dictionary of metrics to log. The following keys are populated automatically: "start", "end".
    * @param event.id: (Optional) a unique identifier for the event. If you don't provide one, BrainTrust will generate one for you.
    * @param options Additional logging options
-   * @param options.allowLogConcurrentWithActiveSpan in rare cases where you need to log at the top level separately from an active span on the logger, set this to true.
+   * @param options.allowConcurrentWithSpans in rare cases where you need to log at the top level separately from spans on the logger elsewhere, set this to true.
    * :returns: The `id` of the logged event.
    */
   public log(
     event: Readonly<StartSpanEventArgs>,
-    options?: { allowLogConcurrentWithActiveSpan?: boolean }
+    options?: { allowConcurrentWithSpans?: boolean }
   ): PromiseUnless<IsAsyncFlush, string> {
-    if (!options?.allowLogConcurrentWithActiveSpan) {
-      const checkCurrentSpan = currentSpan();
-      if (
-        checkCurrentSpan instanceof SpanImpl &&
-        checkCurrentSpan.parentObject === this
-      ) {
-        throw new Error(
-          "Cannot run toplevel Logger.log method while there is an active span. To log to the span, use Span.log"
-        );
-      }
+    if (this.calledStartSpan && !options?.allowConcurrentWithSpans) {
+      throw new Error(
+        "Cannot run toplevel Logger.log method while using spans. To log to the span, use Span.log"
```

Comment:
I think we should recommend that they call `startSpan()` and then log with `span.log()`

Reply:
Updated.